app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
+  before_action :redirect_users_to_home
+
+  def front
```

Comment:
There are 2 ways of writing this - 1) you don't write it at all or 2) you do `def front; end`